examples/camera/camera_coloured_ball_detect.py

```diff
@@ -0,0 +1,44 @@
+from os import environ
+from pitopcommon.current_session_info import get_first_display
+from pitop.camera import Camera
+from pitop.processing.algorithms import BallDetector
+from signal import pause
+from pitop.processing.utils.vision_functions import import_opencv
+
+
+cv2 = import_opencv()
```

Comment:
Probably don't want to encourage this for users who can just import it directly.